### PR TITLE
Add Byteman and btm script to Indy image during build for JDK8 variant.

### PR DESCRIPTION
**Pull request by whitingjr**
This PR looks to add Byteman and a script to the container image.
During image creation Byteman is downloaded, the jar extracted and relocated in the thirdparty directory.
Also added is a Byteman script.

### CODE REVIEW

File: indy/SetStatement300SecondTimeout.btm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Rules to set a timeout on a Statement reference
+
+RULE Timeout set during Connection.createStatement 
+INTERFACE java.sql.Connection
+METHOD createStatement
+AT RETURN
+IF TRUE
+DO System.out.println("BYTEMAN:set 300s timeout during createStatement");$!.setQueryTimeout(300)
```
Comment (ligangty) [inline indy/SetStatement300SecondTimeout.btm:8]: @whitingjr I'm not quite sure of what this change addresses. Do we have jira or related stuff of the problem we're facing?

Comment (whitingjr) [reply indy/SetStatement300SecondTimeout.btm:8]: @ligangty I added you to a ticket that details the context. I hope this provides sufficient information.

Comment (ligangty) [reply indy/SetStatement300SecondTimeout.btm:8]: @whitingjr Sorry and can you send me that ticket again? I'm not sure if I missed that...

Comment (whitingjr) [reply indy/SetStatement300SecondTimeout.btm:8]: @ligangty I sent a message on Gchat

**Review by ligangty — APPROVED** [review]
LGTM. Not sure if 300s is a little long too but we can keep an eye on it.

**Review by jdcasey — APPROVED** [review]

**Pull request merged**